src/components/Vote.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { patchVoteByVoteId } from "../api";
+
+function Vote({ votes, article_id }) {
+  const [votesChange, setVotesChange] = useState(0);
+
+  const incVote = () => {
+    setVotesChange((currVotes) => currVotes + 1);
+    patchVoteByVoteId(article_id, 1);
+  };
+  const decVote = () => {
+    setVotesChange((currVotes) => currVotes - 1);
+    patchVoteByVoteId(article_id, 1).catch((err) => {
```

Comment:
This patch is for decrementing the votes, at the moment when I click remove I actually increase the vote - what is going wrong here?